# Synthetic-Lethality-computational-prediction-method


We will review all existing synthetic lethality calculation prediction methods.

[We used a citationmachin to extract citation information from the papers.](https://www.citationmachine.net/apa/cite-a-website)

#### Statistical analysis


#### Matrix decomposition
[Huang, J., Wu, M., Lu, F., Ou-Yang, L., &amp; Zhu, Z. (2019, December 24). Predicting synthetic lethal interactions in human cancers using graph regularized self-representative matrix factorization - BMC Bioinformatics. BioMed Central.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3197-3)

[Zamanighomi, M., Jain, S. S., Ito, T., Pal, D., Daley, T. P., &amp; Sellers, W. R. (2019, July 12). Gemini: A variational bayesian approach to identify genetic interactions from combinatorial CRISPR screens - genome biology. BioMed Central.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1745-9)


#### Classical Machine learning 
[Das, S., Deng, X., Camphausen, K., &amp; Shankavaram, U. (2018, July 28). DISCOVERSL: An R package for multi-omic data driven prediction of synthetic lethality in cancers. OUP Academic. ](https://academic.oup.com/bioinformatics/article/35/4/701/5061158)

[Benstead-Hume, G., Chen, X., Hopkins, S. R., Lane, K. A., Downs, J. A., &amp; Pearl, F. M. G. (n.d.). Predicting synthetic lethal interactions using conserved patterns in protein interaction networks. PLOS Computational Biology.](https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1006888) 


#### Simple Mulit layer perceptron


[Wan, F., Li, S., Tian, T., Lei, Y., Zhao, D., &amp; Zeng, J. (1AD, January 1). EXP2SL: A machine learning framework for cell-line-specific synthetic lethality prediction. Frontiers.](https://www.frontiersin.org/articles/10.3389/fphar.2020.00112/full) 



#### Graph Nerual Network

[Long, Y., Wu, M., Liu, Y., Zheng, J., Kwoh, C. K., Luo, J., &amp; Li, X. (2021, February 20). Graph contextualized attention network for predicting synthetic lethality in human cancers. OUP Academic.](https://academic.oup.com/bioinformatics/article/37/16/2432/6145565?login=true)

[Cai, R., Chen, X., Fang, Y., Wu, M., &amp; Hao, Y. (2020, March 28). Dual-dropout graph convolutional network for predicting synthetic lethality in human cancers. OUP Academic. ](https://academic.oup.com/bioinformatics/article/36/16/4458/5813330?login=true)

[Predicting synthetic lethality in human cancers via multi-graph ensemble Neural Network. IEEE Xplore. (n.d.).](https://ieeexplore.ieee.org/abstract/document/9630716/) 


[Wang, S., Xu, F., Li, Y., Wang, J., Zhang, K., Liu, Y., Wu, M., &amp; Zheng, J. (2021, July 12). KG4SL: Knowledge graph neural network for synthetic lethality prediction in human cancers. OUP Academic.](https://academic.oup.com/bioinformatics/article/37/Supplement_1/i418/6319703) 


#### Not yet


[Colic, M., Wang, G., Zimmermann, M., Mascall, K., McLaughlin, M., Bertolet, L., Lenoir, W. F., Moffat, J., Angers, S., Durocher, D., &amp; Hart, T. (2019, August 22). Identifying chemogenetic interactions from CRISPR screens with Drugz - Genome Medicine. BioMed Central.](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0665-3)


[Ansari, A. A., Park, I., Kim, I., Park, S., Ahn, S.-M., &amp; Lee, J.-lyun. (2018, October 3). Genomics of drug sensitivity in bladder cancer: An integrated resource for pharmacogenomic analysis in bladder cancer - BMC Medical Genomics. BioMed Central.](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-018-0406-2) 


[Qi, Y., Ye, P., &amp; Bader, J. S. (2005, December 6). Genetic interaction motif finding by expectation maximization – a novel statistical model for inferring gene modules from synthetic lethality - BMC Bioinformatics. BioMed Central.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-6-288)


[Muellner, M. K., Duernberger, G., Ganglberger, F., Kerzendorfer, C., Uras, I. Z., Schoenegger, A., Bagienski, K., Colinge, J., &amp; Nijman, S. M. B. (2014, April 8). Tops: A versatile software tool for statistical analysis and visualization of combinatorial gene-gene and gene-drug interaction screens - BMC Bioinformatics. BioMed Central.](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-98)

[Benfatto, S., Serçin, &amp;O., Dejure, F. R., Abdollahi, A., Zenke, F. T., &amp; Mardin, B. R. (2021, August 28). Uncovering cancer vulnerabilities by machine learning prediction of synthetic lethality - molecular cancer. BioMed Central.](https://molecular-cancer.biomedcentral.com/articles/10.1186/s12943-021-01405-8)

[Benstead-Hume, G., Chen, X., Hopkins, S. R., Lane, K. A., Downs, J. A., &amp; Pearl, F. M. G. (n.d.). Predicting synthetic lethal interactions using conserved patterns in protein interaction networks. PLOS Computational Biology.](https://journals.plos.org/ploscompbiol/article?id=10.1371%2Fjournal.pcbi.1006888) 


[SL2MF: Predicting Synthetic Lethality in Human Cancers via Logistic Matrix Factorization
CSDL: IEEE Computer Society. CSDL | IEEE Computer Society. (n.d.).](https://www.computer.org/csdl/journal/tb/2020/03/08684286/1kepIz6jzDq) 

[Kegel, B. D., Quinn, N., Thompson, N. A., Adams, D. J., &amp; Ryan, C. J. (2021, September 15). Comprehensive prediction of robust synthetic lethality between paralog pairs in cancer cell lines. Cell Systems.](https://www.sciencedirect.com/science/article/pii/S240547122100329X) 


[Li, X.-juan, Mishra, S. K., Wu, M., Zhang, F., &amp; Zheng, J. (2014, April 22). Syn-lethality: An integrative knowledge base of synthetic lethality towards discovery of selective anticancer therapies. BioMed Research International.](https://www.hindawi.com/journals/bmri/2014/196034) 


[Liany, H., Jeyasekharan, A., &amp; Rajan, V. (2019, November 29). Predicting synthetic lethal interactions using heterogeneous data sources. OUP Academic.](https://academic.oup.com/bioinformatics/article/36/7/2209/5646644?login=true) 


[Liany, H., Jeyasekharan, A., &amp; Rajan, V. (2021, January 1). Aster: A method to predict clinically actionable synthetic lethal genetic interactions. bioRxiv.](https://www.biorxiv.org/content/10.1101/2020.10.27.356717v2) 


[Pratapa, A., Balachandran, S., &amp; Raman, K. (2015, June 17). Fast-SL: An efficient algorithm to identify synthetic lethal sets in metabolic networks. OUP Academic.](https://academic.oup.com/bioinformatics/article/31/20/3299/195638) 


[Sinha, S., Thomas, D., Chan, S., Gao, Y., Brunen, D., Torabi, D., Reinisch, A., Hernandez, D., Chan, A., Rankin, E. B., Bernards, R., Majeti, R., &amp; Dill, D. L. (2017, May 31). Systematic discovery of mutation-specific synthetic lethals by mining pan-cancer human primary tumor data. Nature News.](https://www.nature.com/articles/ncomms15580) 


[Tercan, B., Qin, G., Kim, T.-K., Aguilar, B., Kemp, C. J., Chambwe, N., &amp; Shmulevich, I. (2021, January 1). SL-Cloud: A computational resource to support Synthetic Lethal Interaction Discovery. bioRxiv.](https://www.biorxiv.org/content/10.1101/2021.09.18.459450v1.full) 









